__tests__/app.test.js

```diff
@@ -242,6 +262,30 @@ test("400: Sends an appropriate status and error message when given an invalid o
     .then(({ body }) => {
       expect(body.msg).toBe('Invalid order query');
     });
+});
+test("400: Sends an appropriate status and error message when given an invalid topic query", () => {
+  return request(app)
+    .get('/api/articles?topic=hi')
+    .expect(400)
```

Comment:
Probably a 404 status as the topic is "not found" rather than being a bad request. It is a grey area though 😃